packages/magic-trace/magic-trace.v1.0.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
```

Comment:
The version number should not have the `v` prefix. Could you rename it to `1.0.0` instead?